JavaScripit/index.js

```diff
@@ -70,39 +70,29 @@ messageForm.addEventListener('submit', function (event) {
     messageForm.reset();
 });
 
-let githubRequest = new XMLHttpRequest();
 var githubUsername = "Haregebezu";
-const url = `https://api.github.com/users/
-${githubUsername}/repos`;
+const url = `https://api.github.com/users/${githubUsername}/repos`;
+var projects = document.getElementById("projects");
```

Comment:
```suggestion
const projects = document.getElementById("projects");
```

prefer `const`, or `let` if it needs to change.